android/src/main/java/org/mozilla/mozstumbler/client/serialize/ObservationPointSerializer.java

```diff
@@ -48,7 +49,7 @@
     private static final String MLS_NAME = "MLS";
     private static final String ICON_RED_CIRCLE = "http://maps.google.com/mapfiles/kml/shapes/placemark_circle_highlight.png";
     private static final String STYLE_NAME_RED_CIRCLE = "redcircle";
-    private static final String ICON_ARROW = "http://maps.google.com/mapfiles/kml/shapes/arrow.png";
+    private static final String ICON_ARROW = "http://earth.google.com/images/kml-icons/track-directional/track-0.png";
```

Comment:
Hrm.  These links are probably not ok.  We probably don't want to be loading images from the google domain at all.